astropy/modeling/bounding_box.py

```diff
@@ -460,11 +484,14 @@ def _evaluate_model(self, evaluate: Callable, valid_inputs, valid_index,
         valid_outputs = evaluate(valid_inputs)
         valid_outputs_unit = self._get_valid_outputs_unit(valid_outputs, with_units)
 
-        return self.prepare_outputs(valid_outputs, valid_index,
-                                    input_shape, fill_value), valid_outputs_unit
+        return (
+            self.prepare_outputs(valid_outputs, valid_index, input_shape, fill_value),
+            valid_outputs_unit,
+        )
 
-    def _evaluate(self, evaluate: Callable, inputs, input_shape,
-                  fill_value, with_units: bool):
+    def _evaluate(
+        self, evaluate: Callable, inputs, input_shape, fill_value, with_units: bool
+    ):
```

Comment:
Don't know much about black formatting. It looks like both this and the syntax on line 448 are allowed?